sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormLine.cs

```diff
@@ -35,6 +35,11 @@ internal FormLine(FieldBoundingBox boundingBox, int pageNumber, string text, IRe
         /// </summary>
         public IReadOnlyList<FormWord> Words { get; }
 
+        /// <summary>
+        /// An object representing the appearance of the text line.
+        /// </summary>
+        public TextAppearance Appearance { get; }
```

Comment:
does the appearance doc need to go in the FormLine docstring as well?

Reply:
Ohh good point. I will add